src/components/Header/Header.tsx

```diff
@@ -0,0 +1,118 @@
+
+  const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  return (
+    <AppBar position="static" sx={{ bgcolor: "palette.primary" }}>
+      <Container maxWidth="xl">
+        <Toolbar disableGutters>
+          <Box sx={{ flexGrow: 1, display: { xs: "flex", md: "none" } }}>
+            <IconButton
+              size="large"
+              aria-label="account of current user"
+              aria-controls="menu-appbar"
+              aria-haspopup="true"
+              onClick={handleOpenNavMenu}
+              color="inherit"
+            ></IconButton>
+            <Menu
+              id="menu-appbar"
+              anchorEl={anchorElNav}
+              anchorOrigin={{
+                vertical: "bottom",
+                horizontal: "left",
+              }}
+              keepMounted
+              transformOrigin={{
+                vertical: "top",
+                horizontal: "left",
+              }}
+              open={Boolean(anchorElNav)}
+              onClose={handleCloseNavMenu}
+              sx={{
+                display: { xs: "block", md: "none" },
+              }}
+            >
+              <MenuItem
+                onClick={() => {
+                  setAnchorElNav(null);
+                  navigate("/");
+                }}
+              >
+                <Typography textAlign="center">Home</Typography>
+              </MenuItem>
+              <MenuItem
+                onClick={() => {
+                  setAnchorElNav(null);
+                  navigate("/task1");
+                }}
+              >
+                <Typography textAlign="center">Task 1</Typography>
+              </MenuItem>
+              <MenuItem
+                onClick={() => {
+                  setAnchorElNav(null);
+                  navigate("/create");
+                }}
+              >
+                <Typography textAlign="center">Create file</Typography>
+              </MenuItem>
+            </Menu>
+          </Box>
+          <Box sx={{ flexGrow: 1, display: { xs: "none", md: "flex" } }}>
+            <Button
+              onClick={() => {
+                setAnchorElNav(null);
+                navigate("/");
+              }}
+              sx={{ my: 2, color: "white", display: "block" }}
+            >
+              Home
+            </Button>
+            <Button
+              onClick={() => {
+                setAnchorElNav(null);
+                navigate("/task1");
+              }}
+              sx={{ my: 2, color: "white", display: "block" }}
+            >
+              Task 1
+            </Button>
+            <Button
+              onClick={() => {
+                setAnchorElNav(null);
+                navigate("/create");
+              }}
+              sx={{ my: 2, color: "white", display: "block" }}
+            >
+              Create file
```

Comment:
I believe it shouldn't be a part of global header since it's related only to task 1 page
Let's place it there instead